libraries/chain/account_evaluator.cpp

```diff
@@ -280,12 +273,6 @@ void_result account_update_evaluator::do_evaluate( const account_update_operatio
       FC_ASSERT( !o.extensions.value.owner_special_authority.valid() );
```

Comment:
Please remove the commented-out code starting on line 216.

Reply:
Done as part of 1743